docs/Usage/CellMarkers.md

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
I believe a better file would be `Cells.md` with 3 sub sections: What Are Cells, How To Create Them (Currently Cell Markers), and Example Usages (Code, Markdown and Raw as sub sections with screenshots of how the results look inline).
**Note:** This needs to be as if someone has no idea what a jupyter notebook even is and thus should not mention what exporting cells look like but rather how the results look with cells.

Reply:
@wadethestealth Please check the latest commit and see if that works